sdk/cognitivelanguage/azure-ai-language-conversations/samples/async/sample_analyze_orchestration_app_luis_response_async.py

```diff
@@ -31,44 +32,61 @@ async def sample_analyze_orchestration_app_luis_response_async():
     from azure.core.credentials import AzureKeyCredential
 
     from azure.ai.language.conversations.aio import ConversationAnalysisClient
-    from azure.ai.language.conversations.models import ConversationAnalysisOptions
+    from azure.ai.language.conversations.models import LUISTargetIntentResult
 
     # get secrets
-    conv_endpoint = os.environ["AZURE_CONVERSATIONS_ENDPOINT"]
-    conv_key = os.environ["AZURE_CONVERSATIONS_KEY"]
-    orchestration_project = os.environ["AZURE_CONVERSATIONS_WORKFLOW_PROJECT"]
-
-    # prepare data
-    query = "book me a flight ticket to Bali",
-    input = ConversationAnalysisOptions(
-        query=query
-    )
+    clu_endpoint = os.environ["AZURE_CLU_ENDPOINT"]
+    clu_key = os.environ["AZURE_CLU_KEY"]
+    project_name = os.environ["AZURE_CLU_ORCHESTRATION_PROJECT_NAME"]
+    deployment_name = os.environ["AZURE_CLU_ORCHESTRATION_DEPLOYMENT_NAME"]
 
     # analyze query
-    client = ConversationAnalysisClient(conv_endpoint, AzureKeyCredential(conv_key))
+    client = ConversationAnalysisClient(clu_endpoint, AzureKeyCredential(clu_key))
     async with client:
-        result = await client.analyze_conversations(
-            input,
-            project_name=orchestration_project,
-            deployment_name='production',
+        query = "Reserve a table for 2 at the Italian restaurant"
+        result = await client.analyze_conversation(
```

Comment:
It looks like in each of these samples, that the request inputs really don't change - but depending on how the project is configured/orchestrated, you could get a variety of outputs back?

Reply:
not exactly
you need to select what type of project you're targeting (either conversation or orchestration)
in case of conversation, the output is always the same
but in case of orchestration, the output differs depending on best matching project (hence the name orchestration)